### DISCUSSION TITLE
Should Github remove the Trending tab?

### DISCUSSION THREAD

**Discussion by hwmatthewa** [community/community#31657] [Other Feature Feedback, Questions, & Ideas]
For me the trending tab is extremely useful for finding newly released projects or projects which are getting attention that may be useful to my work. It's also a great way to keep tabs on what types of technologies are popular in specific languages.

Do you all think that this tab should be removed?

**Comment by LifeIsStrange** [comment]
Should github be made completely useless?
No.
Next topic?

**Reply by BOSS-3659** [reply]
maybe let me make my computer reparation, a girl drope 2stage on the ground and just my window is broken

**Reply by LifeIsStrange** [reply]
Did she jump after reading about github deleting github?

**Comment by xqm32** [comment]
Only if a more powerful Trending tab is provided, can this one be removed😡

**Reply by LifeIsStrange** [reply]
We don't need some powerful AI bullshit, the current likes by popularity just works and are the true signal. We don't need to reinvent the wheel yet again.
